### PR TITLE
Add tote bag to benefits in Australia

### PR DESCRIPTION
**Pull request by LAKSHMIRPILLAI**
## What are you doing in this PR?
Add Tote bag to benefits in Australia 

[**Trello Card**](https://trello.com/c/AyN6IYu4/1237-🇦🇺-tote-bag-added-to-benefits-in-australia-engineering)

## Why are you doing this?
Copy change on checkout - only in AUS
Copy is:
Limited-edition Guardian Australia tote bag. Offer ends 31 May
Context:
Australia 10 years

## Is this an AB test?
- [ ] Yes
- [ x] No

## Before change(Below Threshold)

image

## Before change(Above Threshold)
image

## After change(Below Threshold -for mobile)

image

## After change(Above Threshold -for mobile)

image

## After change(Below Threshold -for tablet and desktop)

image

## After change(Above Threshold -for tablet and desktop)

image

### CODE REVIEW

File: support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListData.tsx
Comment (i-hardy) [inline support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListData.tsx:19]: Better to use a colour from our [brand palette](https://guardian.github.io/csnx/?path=/story/source-foundations_palette--palette) than just the generic CSS red. Has there been any design input on this at all?

File: support-frontend/stories/checkoutLayout/CheckoutBenefitsList.stories.tsx
Comment (i-hardy) [inline support-frontend/stories/checkoutLayout/CheckoutBenefitsList.stories.tsx:13]: It looks like the tab characters in this file have been replaced with double spaces, it would be good to undo this. If you get Prettier and ESLint set up properly in your editor that should be able to auto-fix it for you.

**Review by i-hardy — APPROVED** [review]
This should be good to merge once we have sign-off on making the offer public, although I think if they have time it would be best to at least have a designer take a look at it first.

**Review by LAKSHMIRPILLAI — COMMENTED** [review]
There were some changes to the copy.Like keep the red text in the same line as benefit for mobile devices and  move the red text to the line below for desktops.Also updated the red text colour to   news.500 or  #FF5943 from brand palette as in the [figma file](https://www.figma.com/file/MQu0jaD3GR1C0CpdLLozOJ/MVP---optimisation---Q4?type=design&node-id=1711-92257&t=4NBzTho1bjwnhMbr-0)

**Review by GHaberis — APPROVED** [review]
Looks good

**Pull request merged**
